_posts/2024-03-07-cache-refresh.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title:  "A cache refresh for Trino"
+author: Manfred Moser
+image: /assets/blog/trino-cache-refresh.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Thinking about our recent work on caching in Trino reminds me of the famous
+saying, ["There are only two hard things in computer science: cache invalidation
+and naming things](https://www.karlton.org/2017/12/naming-things-hard/)." Well,
+in the Trino community we know all about caching and naming. With the recent
+[Trino 439 release]({{site.url}}/docs/current/release/release-439.html), caching
+from object storage file systems got a refresh. Catalogs using the Delta Lake,
+Hive, and Iceberg connectors now get to access performance benefits from the new
```

Comment:
We mention Hudi below. Should we add "(and soon Hudi)" or something?

Reply:
yes .. done